src/main/resources/application.properties

```diff
@@ -1,3 +1,5 @@
+server.servlet.context-path=/hospital-services
```

Comment:
use underscore instead of dash

Reply:
the path has been replaced with underscore